tests/fixtures/module-unification-addon/tests/dummy/src/main.js

```diff
@@ -0,0 +1,14 @@
+import Application from "@ember/application";
+import Resolver from "./resolver";
+import loadInitializers from "ember-load-initializers";
+import config from "../config/environment";
+
+const App = Application.extend({
+  modulePrefix: config.modulePrefix,
+  podModulePrefix: config.podModulePrefix,
+  Resolver
+});
+
+loadInitializers(App, config.modulePrefix + "/src/init");
```

Comment:
Per the fallback setup instructions at https://github.com/rwjblue/ember-module-migrator#running-module-unification-with-fallback-to-classic-app-layout another `loadInitializers` line is needed here that loads addon initializers in the `app` folder.

Reply:
This same addition is needed at https://github.com/ember-cli/ember-cli/blob/e9a7b587a1ebd063985816330fd1a193652cec54/blueprints/module-unification-app/files/src/main.js#L12

Reply:
Added